utilities/backupable/backupable_db_test.cc

```diff
@@ -640,7 +640,8 @@ class BackupableDBTest : public testing::Test {
     // Create logger
     DBOptions logger_options;
     logger_options.env = db_chroot_env_.get();
-    CreateLoggerFromOptions(dbname_, logger_options, &logger_);
+    CreateLoggerFromOptions(dbname_, logger_options, &logger_)
```

Comment:
Why not `EXPECT_OK` ?

Reply:
At least on my Mac, this fails regularly:
 NotFound: /private/tmp/rocksdbtest-501/backupable_db_14615238825541450748/tempdb: No such file or directory
Which is why I did not use EXPECT_OK.  I think there is another bug with the test, and I did not want to dig into it.

Added a comment stating that issue.